building_sim_plugins/building_sim_ign/src/door.cpp

```diff
@@ -332,12 +341,14 @@ class IGNITION_GAZEBO_VISIBLE DoorPlugin
   }
 };
 
-IGNITION_ADD_PLUGIN(DoorPlugin,
-                    System,
-                    DoorPlugin::ISystemConfigure,
-                    DoorPlugin::ISystemPreUpdate)
+IGNITION_ADD_PLUGIN(
+  DoorPlugin,
+  System,
+  DoorPlugin::ISystemConfigure,
+  DoorPlugin::ISystemPreUpdate)
 
 // TODO would prefer namespaced
-IGNITION_ADD_PLUGIN_ALIAS(DoorPlugin,
-                          "door")
+IGNITION_ADD_PLUGIN_ALIAS(
```

Comment:
I wonder why in this case it was not changed to just be a single line since it would have fit the character count?

Reply:
I do not expect the linter to perform any line width optimizations on its own. It only corrects for indentations

Reply:
Hum can you put it into one line? Then we can merge this